test/functional/test-form-data-wrapper.js

```diff
@@ -24,25 +24,27 @@ describes.realWin('FormDataWrapper', {}, env => {
   describe('entries', () => {
     let nativeEntries;
     let nativeDelete;
+
     const scenarios = [{
       description: 'when native `entries` is not available',
 
       beforeEach() {
-        nativeEntries = FormData.prototype.entries;
-        nativeDelete = FormData.prototype.delete;
+        nativeEntries = env.win.FormData.prototype.entries;
```

Comment:
Is this addition of `env.win` due to you being the first to touch a file in a long time?

Reply:
Maybe, I added `env.win` so the test uses the `FormData` reference local to the iframed test code, instead of being referenced across the iframe boundary. This makes the test more realistic to actual usage, and can prevent bugs.

Reply:
Makes sense to me then. :)